readthedocs/restapi/views/model_views.py

```diff
@@ -33,6 +34,28 @@
 log = logging.getLogger(__name__)
 
 
+class PlainTextBuildRenderer(BaseRenderer):
+
+    """
+    Custom renderer for text/plain format.
+
+    charset is 'utf-8' by default.
+    """
+
+    media_type = 'text/plain'
+    format = 'txt'
+
+    def render(self, data, accepted_media_type=None, renderer_context=None):
+        renderer_context = renderer_context or {}
+        response = renderer_context.get('response')
+        if response.status_code != status.HTTP_200_OK:
```

Comment:
Still here can be `None`, I better deep in the docs to see when `renderer_context` is `None`

Reply:
From the docs http://www.django-rest-framework.org/api-guide/renderers/#renderer_contextnone

Looks like this value is always passed here.